.travis.yml

```diff
@@ -75,7 +77,12 @@ jobs:
       - CHROMATIC_APP_CODE="m1dh5kc7oj"
     deploy:
       - provider: script
-        script: npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN && yarn lerna publish --yes --force-publish="*" --canary --preid next --dist-tag next
+        script:
+          - npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN
+          - yarn lerna publish --yes --force-publish="*" --canary --preid next --dist-tag next
+          - git rev-parse --short HEAD | read sha
+          - node utils/get-lerna-version.js | read version
+          - curl -X POST --data-urlencode "payload={\"text\": \"*New canary build published from <https://github.com/Workday/canvas-kit/commit/$sha|merge commit $sha> (v$version)*\n\n\`yarn add @workday/canvas-kit-{module}@$version\`\nor\n\`yarn add @workday/canvas-kit-{module}@next\`\"}" $SLACK_WEBHOOK
```

Comment:
Results in:

**New canary build published from [merge commit 6ead127](https://github.com/Workday/canvas-kit/commit/6ead127) (v3.3.2)**
`yarn add @workday/canvas-kit-{module}@3.3.2`
or
`yarn add @workday/canvas-kit-{module}@next`

Reply:
`SLACK_WEBHOOK` added to Travis environment variable settings.